### PR TITLE
Drop support for ember less than 3.28

### PR DESCRIPTION
**Pull request by leoeuclids**
## Why?
V2 addons only support `ember >= 3.28`, so the support for anything below that should be dropped.
This PR is separate to from #508 to be featured on the 5.0.0 release changelog.

## What it achieves
Allows the conversion into a V2 addon (In progress #508).

## What this PR does
Changes the ember-try scenarios, dropping `ember-lts-2.4`, as well as adding `ember-lts-4.4` and `ember-lts-4.8`.
Updates the Readme compatibility table with the 5.0.0 version.

- [x] Remove `ember-lts-3.24` scenario
- [x] Add `ember-lts-4.4` scenario
- [x] Add `ember-lts-4.8` scenario
- [x] Update Readme

### CODE REVIEW

**Review by NullVoxPopuli — APPROVED** [review]
LGTM

**Comment by leoeuclids** [comment]
A11y tests should pass now.

**Comment by leoeuclids** [comment]
Third try's the charm.

File: tests/dummy/app/templates/index.hbs [hunk 1]
```diff
@@ -75,7 +75,7 @@
       <table>
         <thead>
           <tr>
-            <th></th>
+            <th>&vArr;</th>
```
Comment (NullVoxPopuli) [inline tests/dummy/app/templates/index.hbs:78]: Why this change?

Comment (leoeuclids) [reply tests/dummy/app/templates/index.hbs:78]: This is to fix the A11Y failing test. I'm not sure if there is a better way to fix it.

Comment (NullVoxPopuli) [reply tests/dummy/app/templates/index.hbs:78]: This error?

```
Table header text should not be empty 
```

Why was Item considered empty?

Comment (leoeuclids) [reply tests/dummy/app/templates/index.hbs:78]: Yes, that one.

**Comment by leoeuclids** [comment]
Thanks for merging!
